stdlib/codecs.pyi

```diff
@@ -115,16 +115,16 @@ def iterdecode(iterator: Iterable[bytes], encoding: str, errors: str = ...) -> G
 if sys.version_info >= (3, 10):
     def unregister(__search_function: Callable[[str], CodecInfo | None]) -> None: ...
 
-BOM: bytes
-BOM_BE: bytes
-BOM_LE: bytes
-BOM_UTF8: bytes
-BOM_UTF16: bytes
-BOM_UTF16_BE: bytes
-BOM_UTF16_LE: bytes
-BOM_UTF32: bytes
-BOM_UTF32_BE: bytes
-BOM_UTF32_LE: bytes
+BOM: bytes  # can be `BOM_UTF16_LE` or `BOM_UTF16_BE`, depends on `sys.byteorder`
```

Comment:
Any reason not to type it as `Literal[b'BOM_UTF16_LE', b'BOM_UTF16_BE']`?

Reply:
Didn't think of that! Thank you!